README.md

```diff
@@ -119,29 +121,14 @@ In turn this enables [Wasm components to be able to import][]
 
 ### Other Module Types
 
-Other module types may define their own host module sources. If no module phase import
-is defined, it will fail during the loading phase.
+Any other host-defined module types may define their own host module sources. If no module phase import is defined, it will fail during the loading phase.
```

Comment:
This sentence doesn't seem clear. Maybe:

```suggestion
Any other host-defined module types may define their own host module sources. If a given module does not define a source representation for it's source, importing it with a "source" phase target fails.
```

Reply:
Sounds good, thanks!